### DISCUSSION TITLE
How to reuse Nova 4 core fields/components in custom tools/resource tools?

### DISCUSSION THREAD

**Discussion by aliwesome** [laravel/nova-issues#6475] [Q&A] [answered]
I want to reuse Nova core components on my custom tools, I tried it with multiple approaches but I didn't succeed. 

It was pretty easy on version 3, is there any guide for that?

For example I want to reuse `<MultiSelectField />` in my tool, so I can get its benefits and not including other libraries like `vue-multiselect` on my custom tool.

**Comment by crynobone** [comment, marked as answer]
We don't recommend using any of the component unless explicitly documented in the official documentation as those may/will contain breaking changes between minor and patch releases.

**Reply by aliwesome** [reply]
As the project maintainer, it is crucial that our dashboard custom tools remain synchronized with the core components. This synchronization is essential because we utilize the same components throughout the entire project. To manage potential breaking changes effectively, it is important to update our tools in tandem with core updates. This approach minimizes both the time and programming effort required, ensuring a more efficient development process.
